test/powershell/engine/Help/HelpSystem.Tests.ps1

```diff
@@ -181,6 +181,26 @@ Describe "Validate that Get-Help returns provider-specific help" -Tags @('CI', '
     }
 }
 
+Describe "Get-Help should find help info within help files" -Tags @('CI', 'RequireAdminOnWindows') {
+    It "Get-Help should find help files under pshome" {
+        $helpFile = "about_testCase.help.txt"
+        if (!(Test-Path $PSHOME\en-US))
+        {
+            New-Item -ItemType Directory -Path "$PSHOME\en-US" -ErrorAction SilentlyContinue
+        }
+        
+        try
+        {
+            New-Item -ItemType File -Path "$PSHOME\en-US\$helpFile" -Value "about_test" -ErrorAction SilentlyContinue
```

Comment:
If we use `SilentlyContinue` we do not need to use `Test-Path`.
We can move New-Item's in `BeforeAll`:
```powershell
BeforeAll {
    $helpFile = "about_testCase.help.txt" 
    New-Item -ItemType Directory -Path "$PSHOME\en-US" -ErrorAction SilentlyContinue
    New-Item -ItemType File -Path "$PSHOME\en-US\$helpFile" -Value "about_test" -ErrorAction SilentlyContinue
}
```